### PR TITLE
adding 2.11.0

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

File: cli-feed-v3.json [hunk 1]
```diff
@@ -404,6 +404,45 @@
...
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/2.0.0-10288.zip",
+      "sha2": "DB1A6D8A20E600D57824F03EC1431D2169C26A952D6BE2FC815EAC35CEC4EE0D",
+      "FUNCTIONS_EXTENSION_VERSION": "~2",
+      "requiredRuntime": ".NET Core",
+      "minimumRuntimeVersion": "2.1",
+      "standaloneCli": [
+        {
+          "OperatingSystem": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.1.748/Azure.Functions.Cli.osx-x64.2.2.1.zip",
```
Comment (soninaren) [inline cli-feed-v3.json]: Download link is incorrect
https://functionscdn.azureedge.net/public/2.2.1/Azure.Functions.Cli.osx-x64.2.2.1.zip

Comment (soninaren) [reply cli-feed-v3.json]: For all versions of standalone cli's

Comment (brettsam) [reply cli-feed-v3.json]: Argh. Yeah I'll fix it.

File: cli-feed-v3.json [hunk 2]
```diff
@@ -404,6 +404,45 @@
...
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.1.748/Azure.Functions.Cli.win-x86.2.2.1.zip",
+          "sha2": "DB1A6D8A20E600D57824F03EC1431D2169C26A952D6BE2FC815EAC35CEC4EE0D"
+        },
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.1.748/Azure.Functions.Cli.linux-x64.2.2.1.zip",
```
Comment (soninaren) [inline cli-feed-v3.json]: This link is not working for some, is there a typo or missing upload?

File: cli-feed-v3.json [hunk 3]
```diff
@@ -404,6 +404,45 @@
...
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.2.1/Azure.Functions.Cli.win-x86.2.2.1.zip",
+          "sha2": "DB1A6D8A20E600D57824F03EC1431D2169C26A952D6BE2FC815EAC35CEC4EE0D"
+        },
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.2.1/Azure.Functions.Cli.linux-x64.2.2.1.zip",
```
[the remark this answers is not in the document]
Comment (soninaren) [reply cli-feed-v3.json]: Looks good

**Pull request merged**